corehq/apps/export/templates/export/partials/bootstrap5/table.html

```diff
@@ -257,7 +257,7 @@ <h4 class="modal-title">
                       class="btn btn-outline-primary btn-sm"
                       data-bind="visible: emailedExport.canUpdateData,
                                  attr: {
-                                   'data-target': '#modalRefreshExportConfirm-' + id() + '-' + emailedExport.groupId()
+                                   'data-bs-target': '#modalRefreshExportConfirm-' + id() + '-' + emailedExport.groupId()
```

Comment:
Does this change is flagged by the migration tool?

Reply:
It originally didn't get flagged because the migration tool was only looking for regular data attributes set directly in elements, like `<div data-target="#myModal">`, but https://github.com/dimagi/commcare-hq/pull/35160 updates the tool to address this.

Reply:
I see. The PR was added later 😅